coda/coda_validate/views.py

```diff
@@ -77,8 +77,8 @@ def items(self, obj):
             # instead, let's do this:
             # http://elpenia.wordpress.com/2010/05/11/getting-random-objects-from-a-queryset-in-django/
             v = validations.filter(
-                last_verified__lte=datetime.datetime.now() - \
-                    settings.VALIDATION_PERIOD
+                last_verified__lte=datetime.datetime.now() -
```

Comment:
So I'm inclined to agree with [PEP8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) on their recommendation to put operators with their operands on the new line so it is easier to tell what is being subtracted in this case. I am not saying _this_ should be changed, but just wanted to throw it out there as a possible suggestion. I haven't been doing this myself but I think I will start to.